src/main/java/ch/njol/skript/Skript.java

```diff
@@ -923,6 +924,18 @@ public void run() {
 			}
 		}, this);
 
+		// Send a warning to console when the plugin is reloaded
+		Bukkit.getPluginManager().registerEvents(new Listener() {
+			@EventHandler
+			public void onServerReload(ServerLoadEvent event) {
+				if (!(event.getType() == ServerLoadEvent.LoadType.RELOAD)) return;
+
+				Skript.warning("Looks like you've used /reload or used a plugin to reload Skript. This is not supported behaviour.");
```

Comment:
```suggestion
				Skript.warning("Looks like you've used /reload or used a plugin to reload Skript. This is not supported behaviour.");
```

This should be better worded in my honest opinion, maybe something like:
"Hey! It looks like you've used the /reload command or used a plugin to reload Skript. This is not supported and will cause issues!"

Reply:
I figured it out xd my bad